internal/pkg/service/buffer/store/schema/schema_test.go

```diff
@@ -100,6 +100,26 @@ func TestSchema(t *testing.T) {
 			}).Key(),
 			"secret/export/token/123/my-receiver/my-export",
 		},
+		{
+			s.Runtime().Prefix(),
+			"runtime/",
+		},
+		{
+			s.Runtime().Workers().Prefix(),
+			"runtime/workers/",
+		},
+		{
+			s.Runtime().Workers().Active().Prefix(),
+			"runtime/workers/active/",
+		},
+		{
+			s.Runtime().Workers().Active().IDs().Prefix(),
+			"runtime/workers/active/ids/",
+		},
+		{
+			s.Runtime().Workers().Active().IDs().Node("my-node").Key(),
+			"runtime/workers/active/ids/my-node",
+		},
```

Comment:
Pridal som novy prefix, do ktoreho sa budu ukladat aktivne nodes.

Funguje to tak, ze dana `node` sa zaregistruje pomocou PUT operacie.
S nastavenou `lease`.

`Lease` zabezpecuje to, ze kluc sa automaticky zmaze, ked sa klient odpoji a prejde `TTL` (`15s`).

Okrem toho, pri node shutdown zavolame aj manualne DELETE operaciu.

Ostatne nodes maju nastaveny `watcher` na tento prefix a tak vedia o zmenach.